README.md

```diff
@@ -104,7 +104,7 @@ Want to get started with a step by step tutorial for contributing, using differe
 <details>
 <summary><b>Click to expand tutorial list</b></summary>
 
-- [Getting started: GitHub (In your Browser) - (Easy)](tutorials/(Easy)&#32;GitHub&#32;(In&#32;your&#32;Browser).md)
+- [Getting started: GitHub (In your Browser) - (Easy)](tutorials/README.md)
```

Comment:
The "README.md" file in ***/tutorials*** should serve as a front page, and contains the list to the tutorials + links. You should put your tutorial in a separate file in this same folder and name it according to the git,method & difficulty level to the tutorial you created.
e.g: "**GitHub_In_Your_Browser_(Easy).md**"

Reply:
Hi, just pushed another change for this

Reply:
Alright, seems in order. The PR shall be merged and I will setup a basic structure/link ref on the folder's readme so it's easy for people to check. Good.